wrapper.js

```diff
@@ -109,6 +109,11 @@ function setupMethods (soljson) {
       });
     }
 
+    // Forward compatibility with 0.6.x
+    if (typeof readCallback === 'object') {
```

Comment:
At this point it shouldn't be an object, right? The only case that would support it is `compileStandard` which would have returned in the first lines

Reply:
The first line in `compilerStandardWrapper` is a short circuit to `compileStandard` if it exists. Handling of the callback is done there.

Reply:
We could move this single if there to avoid duplication of it, but not sure there is any point adding the smt callback into the wrapper anyway (as it will never work) so perhaps it makes more sense keeping the PR as is anticipating the smt changes will be merged.

Reply:
Ah sorry, confused it with `solidity_compile`

Reply:
But still, this condition would never be true, since the only case that actually accepts an object with callbacks (`solidity_compile`) would lead to `compileStandard` existing

Reply:
Ah, or would that be for new solc-js + old solc?

Reply:
> Ah, or would that be for new solc-js + old solc?

Yes